<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <!-- yolo -->
  <include file="$(find darknet_ros)/launch/darknet_ros_test.launch" />
  <!-- usb cam  -->
 <!-- <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param n>-->

  <!--시각화를 위해 넣어놓은거니깐 필요없으면 주석처리-->
  <!-- <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node> -->

<!--main python파일도 같이 실행 -->
<node pkg="vision_distance" type="track_mission_lidar.py" name="track_mission"/>

<!--<node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find obstacle_detector)/resources/obstacle_detector.rviz"/> -->
 

</launch>
